### ISSUE TITLE
Display unified diff when resolving edit conflicts

### ISSUE THREAD

**Issue by sbrl** [sbrl/Pepperminty-Wiki#64]
A unified diff would be very helpful when resolving conflicts, as it allows the user to see what has changed. Unfortunately, this is not a trivial task without adding a bunch of extra files and a ton of weight to Pepperminty Wiki. Xdiff doesn't work with PHP 7, and other implementations add a *ton* of files and would have to be completely rewritten.

On the other hand, we could do the diff on the client side with javascript. The library [jsdiff](https://github.com/kpdecker/jsdiff) looks awesome for this purpose. Currently we are waiting on a response from kpdecker/jsdiff#106, as it would be preferable to utilise [RawGit](https://rawgit.com/)'s production CDN to deliver the file, as cdnjs doesn't have the library, and JSDelivr's copy is even more outdated than github pages.

**Comment by sbrl** [comment]
I've opened cdnjs/cdnjs#7750 to request that jsdiff be added to cdnjs.

**Comment by sbrl** [comment]
Cdnjs has been kind enough to add jsdiff to their CDN! Apparently the issue I opened to request jsdiff addition was a duplicate of cdnjs/cdnjs#7127. We can now add a unified diff to the conflict resolution page!

**Comment by sbrl** [comment]
Done.
